packages/streaming-image-volume-loader/src/helpers/sortImageIdsAndGetSpacing.ts

```diff
@@ -126,11 +126,22 @@ export default function sortImageIdsAndGetSpacing(
       Math.floor(imageIds.length / 2);
   }
 
-  const { imagePositionPatient: origin } = metaData.get(
+  const { imagePositionPatient: origin, sliceThickness } = metaData.get(
     'imagePlaneModule',
     sortedImageIds[0]
   );
 
+  // if zspacing = 0, use sliceThickness if possible
+  if (zSpacing === 0) {
+    if (sliceThickness) {
+      console.log('Could not calculate zSpacing. Using sliceThickness');
+      zSpacing = sliceThickness;
+    } else {
+      console.log(
+        'Could not calculate zSpacing. The VolumeViewport is compromised.'
```

Comment:
Set zSpacing to 1 - not a great option here, but at least it will render.

Reply:
agree

Reply:
Done